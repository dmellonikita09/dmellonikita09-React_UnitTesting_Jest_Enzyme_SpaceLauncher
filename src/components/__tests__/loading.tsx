import React from 'react';
import { mount } from 'enzyme';
import Loading from '../loading';

describe('Loading', () => {
  it('should render correctly', () => {
    const wrapper = mount(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});