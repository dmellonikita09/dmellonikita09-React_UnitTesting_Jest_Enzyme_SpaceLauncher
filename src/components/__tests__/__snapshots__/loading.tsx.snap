// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render correctly 1`] = `
<Styled(Component)>
  <ForwardRef(SvgLogo)
    className="css-kahotv"
  >
    <svg
      className="css-kahotv"
    >
      logo.svg
    </svg>
  </ForwardRef(SvgLogo)>
</Styled(Component)>
`;
